server/test/routes/auth.test.js

```diff
@@ -22,6 +22,7 @@ const tempUser = {
     password: '11qQw@123',
   },
 };
+const authTest = () => {
```

Comment:
you don't need this assignment, please remove it

Reply:
I need to be more clear, is it the 
`const authTest = ()`
or the entire function assignment
`const authTest = () => {`